portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsWebContent.testcase

```diff
@@ -25,6 +25,86 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-184580 - StaleObjectExcpetion when changing a web content portlet's displayed article in a publication."
```

Comment:
For the description, we usually describe the purpose of the test instead of the title of the ticket, can you please update it? You can update it like `Assert no error throws when changing a web content portlet's displayed article in a publication.`